README.md

```diff
@@ -8,9 +8,8 @@
 ## Typical Usage
 
 ##### Using Docker Compose
-```
-docker-compose up -d
-```
+1. Add the service using the following template [docker-compose.yml](https://github.com/realies/soulseek-docker/blob/master/docker-compose.yml)
+2. Start the service `docker-compose up -d --no-deps --build soulseek`
```

Comment:
Would it be necessary to link to the template in 1. as part of the usage docs? Additionally, what would be the benefit of adding `--no-deps --build soulseek` when the proposed change for `docker-compose.yml` points to the Docker hub hosted image?

Reply:
Again it's for simplifying things, just click and copy.
The no deps build thing is to make sure that only that service from docker-compose is built. People do usually have multiple services, like plex, amule, stash, etc...

Reply:
@tetreum, ok, I agree with naming the container. I am still not sure why we need --no-deps and --build soulseek, when the target compose file only contains soulseek, which already has an image on Docker Hub.

From the docs https://docs.docker.com/compose/reference/up/:
--no-deps                  Don't start linked services.
--build                    Build images before starting containers.